CONTRIBUTING.md

```diff
@@ -14,3 +14,223 @@ If you are sending a pull request and it's a non-trivial change beyond fixing ty
 [the ICLA (individual contributor license agreement)](https://feedback.line.me/enquete/public/799-Z6ygVz3U). Please
 contact us if you need the CCLA (corporate contributor license agreement).
 
+### How to build
```

Comment:
Maybe we can just remove this section completely? lol

Here's the new README.md:

https://github.com/trustin/armeria/blob/contributor_guide/README.md